tools/Microsoft.AspNet.Server.Kestrel.GeneratedCode/KnownHeaders.cs

```diff
@@ -59,7 +59,7 @@ protected string Term(string name, int offset, int count, string array, string s
             {
                 ulong mask = 0;
                 ulong comp = 0;
-                for (var scan = 0; scan != count; ++scan)
+                for (var scan = 0; scan < count; scan++)
```

Comment:
I think that *is* the generator; so its use is compile time, not necessarily perf sensitive.

I'm coming from the view that its best not to have examples of c++ iterator for-loop syntax in the code base as its not very good for c# for-loops which can have the extra overhead of bounds checking on each iteration if you don't follow the jit's happy pattern. As generations of devs will look to this and the other code bases for inspiration; but also probably not pay close attention to the context.